### DISCUSSION TITLE
On the star's page, add an option to filter by list (or none at all)

### DISCUSSION THREAD

**Discussion by thehale** [community/community#8611] [Other Feature Feedback, Questions, & Ideas]
Lists are an amazing new feature that I think will be phenomenally useful. To help people like myself with lots of stars get organized using these lists, I'd love to have the option to filter my starred repos to see which ones haven't been assigned a list yet. Additionally, while there is the big button to see just the repos in a specific list, being able to filter by that on the main page (e.g. list repos in list A and/or list B) would also be really nice.

**Comment by myesn** [comment]
![image](https://user-images.githubusercontent.com/18598579/145506963-943156a4-dc6e-4b00-bdb8-18f33e98ff24.png)

I am looking forward to this feature very much, because I star too many items, and I do not know which stars are included in Lists and which are not.

**Reply by Vovanchik25071982** [reply]
0x8b1430546Cf6E5c1c83F04e21206834aE54AD09E

**Comment by equal-l2** [comment]
+1 for this

**Comment by anshuld1995** [comment]
hello , i also want a star in project how can i do this .

**Reply by Patstar210** [reply]
I'm not sure but any support at all will help I'm just trying to figure things out

**Comment by tjkirch** [comment]
I'd love this request to be implemented because I'd like to narrow down items in a given star list by another filter, such as programming language.

**Reply by Patstar210** [reply]
Thanks for your input any help is all improvements
